vulnerabilities/data_source.py

```diff
@@ -46,11 +46,18 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclasses.dataclass
+class VulnerabilitySeverity:
+    severity_type: str
```

Comment:
@pombredanne  along the same lines, how about instead of `ScoringSystem` dataclass, we have a `ScoringSystem` django model, and have something to provide intial seed rows of  `ScoringSystems` . 

On the `Severity` dataclass side, we could turn `scoring_system: ScoringSystem` into something like `scoring_system: str` where the `scoring_system` would have value of the identifier.

Reply:
The advantage of this would be all the `ScoringSystem`s would live in db together with the values, giving us nice mapping of `vuln->severity->severity_typ_details` . 

On the other hand when using `ScoringSystem` dataclass, all the ScoringSytems would live in the memory so our mapping would be `vuln->severity` , if I have to know more about the provided `severity` I would need to lookup  the python file containing  `scoring_systems`  .

Reply:
Using objects instead, after thinking more about this. 

Main reason being, having a document instead of objects does not enforce documenting different types of severity types :)